src/test/java/roomescape/reservationtime/acceptance/ReservationTimeAcceptanceTest.java

```diff
@@ -0,0 +1,133 @@
+package roomescape.reservationtime.acceptance;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.lang.reflect.Field;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import roomescape.reservationtime.controller.ReservationTimeController;
+import roomescape.reservationtime.domain.ReservationTime;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+class ReservationTimeAcceptanceTest {
+
+    @Autowired
+    private ReservationTimeController reservationTimeController;
+
+    @Test
+    @DisplayName("시간을 조회하는 API를 요청한다.")
+    void getReservationTimes() {
+        // given
+        var params = Map.of(
+                "startAt", "10:00"
+        );
+
+        RestAssured
+                .given().log().all()
+                .contentType(ContentType.JSON)
+                .body(params)
+                .when().post("/times");
+
+        // when
+        List<ReservationTime> reservationTimes = RestAssured
+                .given().log().all()
+                .when().get("/times")
+                .then().log().all()
+                .statusCode(200).extract()
+                .jsonPath().getList(".", ReservationTime.class);
+
+        // then
+        ReservationTime expected = new ReservationTime(
+                1L,
+                LocalTime.of(10, 0)
+        );
+
+        assertThat(reservationTimes.getFirst()).isEqualTo(expected);
+        assertThat(reservationTimes.size()).isEqualTo(1);
+    }
+
+    @Test
+    @DisplayName("시간을 생성하는 API를 요청한다.")
+    void createReservationTime() {
+        // given
+        var params = Map.of(
+                "startAt", "10:00"
+        );
```

Comment:
사소하지만 이런 케이스는 그냥 한 줄로 선언해도되지 않을까요? 😃 
```suggestion
        var params = Map.of("startAt", "10:00");
```

Reply:
한 줄로 선언해도 괜찮을 것 같긴 하지만, 이후에 파라미터가 추가될 가능성을 고려해 확장성을 염두에 둔 코드 스타일입니다!

사소한 질문이지만, 이런 사소한 엔터 처리(줄바꿈) 같은 경우가 많아질 때 어떤 코드 스타일이 가장 가독성이 좋은지에 대해서는 아직 저만의 확고한 기준을 찾지는 못한 것 같아요ㅎㅎ 로키는 이런 경우 어떤 방식을 주로 사용하시나요?

       // given
        ReservationTime reservationTime = new ReservationTime(
                1L,
                LocalTime.of(10, 0)
        );
        Reservation reservation = new Reservation(
                1L,
                "미소",
                LocalDate.of(2025, 4, 21),
                reservationTime
        );
        reservationTimeRepository.save(reservationTime);
        reservationRepository.save(reservation);

예를 들어. 이런 경우는 save 메서드 안에 바로 객체를 생성해서 넘기는 것도 가능하고 (변수 추출하지 않고)

        ReservationTime found = jdbcTemplate.queryForObject(
                "select id, start_at from reservation_time where id = ?",
                (resultSet, rowNum) ->
                        new ReservationTime(
                                resultSet.getLong("id"),
                                LocalTime.parse(resultSet.getString("start_at"))
                        ),
                1L);

반면 이런 경우에는 각각 변수로 추출해서 작성하는 것도 생각해볼 수 있을 것 같아요.

Reply:
음... 사실 이 방식은 결국 팀적인 기준을 보통 따라가게 되기 때문에 특히 정답은 없는 영역인 것 같습니다. 

제 개인적인 경험을 말씀드리자면, 자바진영에서는 미소처럼 파라미터 한 줄에 하나씩만 위치시키는 방식을 그렇게 선호하지는 않는 느낌이었어요. 반면에 코틀린 같은 경우에는 이렇게 사용하는 것이 더 흔한 것 같았어요. 🤔

이방식 저 방식 다 있는거지만 다만 중요한건 한번 정했다면 일관성있게 가져가야한다는 것이 중요하다고 생각해요. 😃
특히 함께 일하는 입장에서 이런 코드 스타일을 통일하는 것이 쉽지 않기 때문에 나중에는 Lint 같은 것들을 지정하거나 혹은 IDE에 자동 정렬 포맷을 팀의 기준에 맞게 정의해서 동일하게 셋팅해둔다거나? 등등의 방식으로 맞춰가는 것 같아요.

Reply:
> 한 줄로 선언해도 괜찮을 것 같긴 하지만, 이후에 파라미터가 추가될 가능성을 고려해 확장성을 염두에 둔 코드 스타일

자바는 사실 [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) 기능을 지원하지 않기 때문에 미소가 작성하신 방식도 결국 조금은 불편함이 생기더라구요. 😃

저는 코트린으로 코드 짤 때는 미소 스타일로 코드를 작성 +  trailing comma 기능을 적극 활용하여 추후 파라미터 추가 시에도 쉽게 추가할 수 있도록 짜는 편이에요.

Reply:
좋은 인사이트 감사합니다! 코틀린은 다양한 걸 지원하는 군요.. 👀